docs/API.md

```diff
@@ -829,6 +830,7 @@ value. For desired result, configure MSS first.***
 | --------------------- | ----- | ------- | --------- | ------ | -------- | ------------- |
 | `SRTO_RCVLATENCY`     | 1.3.0 | pre     | `int32_t` | msec   | 0        | positive only |
 
+- **NB!** The default [live mode](#transmission-method-live) settings set SRTO_RCVLATENCY to 120 ms!
```

Comment:
Isn't it better to replace `0` in the `Default` column with `See below`?

Reply:
Maybe replace 0 with some word meaning "it depends on other settings". But in one word :)

Reply:
> NB: The default live mode settings set SRTO_RCVLATENCY to 120 ms!

A task for later myself:
- [ ] Replace "**NB!**" with "**NB:**"

Reply:
latency=0 is the default value for file mode. But live mode is the default mode, and it sets 120 ms. So probably the best way is to change default to 120 ms, and in the note describe the File mode configuration.

Reply:
It's complicated to speak about the default, if you have an option that can change default values of multiple other options. The "absolute default" actually is 120ms, but it's only because the "live" mode is default. Therefore it will be misleading. It's best if ALL OPTIONS that are affected by changes in `SRTO_TRANSTYPE` have no described "default" in this table, but rather state this "See below" because the explanation will be longer:

@stevomatthews There can't be such a thing as "NONE default". No default value can only mean that setting particular option is obligatory for the application and no operation can be allowed until it's set. We do have default values for all options, though, just we have "multiple levels of defaults" due to this integration option, `SRTO_TRANSTYPE`.